### DISCUSSION TITLE
Entity Operator - topic-operator connection refused

### DISCUSSION THREAD

**Discussion by Thomas-Rice** [strimzi/strimzi-kafka-operator#6264] [Q&A]
Hi, 
I was wondering if anyone could help me with an issue. I had a previously working installation of Strimzi on my cluster. But recently the topic operator has started failing due it's probe not being able to communicate with the health check http://10.7.219.50:8080/healthy. 
The probe settings are all default and I have tried investigating logs of all the pods in the namespace but I am out of ideas.

![image](https://user-images.githubusercontent.com/9338637/151217045-38e3951d-4b5e-4389-a2ab-286a43f1158f.png)

Additional info from the pod logs:
![image](https://user-images.githubusercontent.com/9338637/151221087-c4b31e02-69aa-4ecb-a456-915d66d9a820.png)

**Comment by antonyjohnpaul23us** [comment]
I'm facing the same issue from yesterday on a cluster that was running fine. Any pointers to this issue would help

![image](https://user-images.githubusercontent.com/18515695/151860389-b1284d1d-39ef-4757-baca-139c3c5607bd.png)
